eaf-browser.el

```diff
@@ -673,6 +674,13 @@ Otherwise send key 'esc' to browser."
   (interactive "M[EAF/browser] URL: ")
   (eaf-open (eaf-wrap-url url) "browser" args))
 
+;;;###autoload
+(defun eaf-open-browser-other-window (url &optional args)
+  "Open EAF browser application given a URL and ARGS in other window."
+  (interactive "M[EAF/browser] URL: ")
+  (other-window 1)
```

Comment:
This needs to verify that there's more than 1 window present.

Reply:
Or make this new window

Reply:
I fix it in the new patch to create a new window